bin/mainflux-launch.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+###
+# Launches all Mainflux Go binaries when they are installed globally.
+# Also launches NATS broker instance, expecting that
+# `gnatsd` is installed globally.
+#
+# Expects that Cassandra is already installed and running.
+#
+# Does not launch NodeJS MQTT service - this one must be launched by hand for now.
+###
+
+# Kill all mainflux-* stuff
+function cleanup {
+	pkill mainflux
+}
+
+gnatsd &
+# Wait a bit for NATS to be on
+sleep 0.1
+mainflux-http &
+mainflux-manager &
+mainflux-writer &
+mainflux-coap &
+
+trap cleanup EXIT
+
+while : ; do sleep 1 ; done
```

Comment:
Why do we need this?

Reply:
This is the testing utility, not used for deployment. If we do not wait and catch CTRL+C signal, then kill all the `mainflux-` processes, then we will have to add a commaned to stop them, or chase them one by one and kill them manually.

I find it very convinient to stay in foreground mode with the script, and then kill all the processes with CTRL+C - this is very useful for fast starting whole system (all services) during development process.